src/pincode/authentication.ts

```diff
@@ -164,10 +164,15 @@ async function getPasswordHashForPin(pin: string) {
   return getPasswordHash(password)
 }
 
-function getPasswordHash(password: string) {
-  return sha256(Buffer.from(password, 'hex')).toString('hex')
+// TODO: this existing implementation implies password is in hex (no '0x' prefix)
+// but we should lift that restriction as it's too easy to misuse
```

Comment:
We can change the behavior in a future PR.
It requires resetting the stored hashes, so existing accounts can still unlock.